Admin/AbstractAdmin.php

```diff
@@ -1978,6 +1985,11 @@ public function getBreadcrumbs($action)
      */
     public function buildBreadcrumbs($action, MenuItemInterface $menu = null)
     {
+        @trigger_error(
+            'The '.__METHOD__.' method is deprecated since version 3.x and will be removed in 4.0 .',
```

Comment:
That it looks like an unfinished version number, but if you're sure, I'll change it.